src/subdataframe/subdataframe.jl

```diff
@@ -130,10 +176,22 @@ index(sdf::SubDataFrame) = index(parent(sdf))
 nrow(sdf::SubDataFrame) = ncol(sdf) > 0 ? length(rows(sdf))::Int : 0
 ncol(sdf::SubDataFrame) = length(index(sdf))
 
+function Base.getindex(sdf::SubDataFrame, colind::ColumnIndex)
+    Base.depwarn("`sdf[colind]` will create a view of `parent(sdf)[colind]` in the future." *
+                 " Use sdf[:, [colind]]` to get a `DataFrame`.", :getindex)
+    return parent(sdf)[rows(sdf), colinds]
+end
+
 function Base.getindex(sdf::SubDataFrame, colinds)
+    Base.depwarn("`sdf[colinds]` will create a `SubDataFrame` in the future." *
+                 " Use `sdf[:, colinds]` to get a `DataFrame`.", :getindex)
     return parent(sdf)[rows(sdf), colinds]
 end
 
+function Base.getindex(sdf::SubDataFrame, rowind::Integer, colind::ColumnIndex)
```

Comment:
This is redundant with the method below, right?

Reply:
I have forgotten to add depwarn in one of them. Fixed